setup.py

```diff
@@ -14,16 +14,15 @@
     long_description=open('README.rst', 'r').read(),
     packages=find_packages(),
     install_requires=[
-        'django>=1.8,<1.9a1',
-        'djangorestframework>=3.1',
+        'django>=1.9,<1.10a1',
+        'djangorestframework>=3.3',
```

Comment:
To the first point, that's great! To the third: if users wish to include resolwe in existing Django project, they already have some requirements and a virtualenv. So, looser is better, because some of their requirements may clash with ours. To the middle point :) It would be appropriate to test all combinations, but almost nobody does that in practice. Most open-source projecs test the latest requirements that match their setup.py. When they get bug reports from the community, they think about solutions: either to fix bugs or increase the requirements. If we would test 2 combinations: the highest versions that match the requirements and the lowest, that would already be pretty impressive. I don't think we have to worry about this much now, but we will have to in the future.